### ISSUE TITLE
Limited documentation on how to change time axis on plots

### ISSUE THREAD

**Issue by bartech3** [ggPMXdevelopment/ggPMX#108]
I had did not find any explicit documentation on how to change the time axis of plots. The following did work 

```r
ctr %>% 
  pmx_plot_cats(
    "npde_time",
    cats = "DOSE",
    aess = list(x="TADQBW"), 
    filter = TADQBW < 24 & TADQBW > -2,
    labels = list(title = "NPDE vs TAD", x="Time after last dose (h)"))
```

**Comment by sehamsick** [comment]
Thank you for the suggestion! Following documentation in the will be added in the next version of ggPMX:
*aess can be used to change time variable within the plot (e.g. aess = list(x="TADQBW"))*

**Comment by tynsci** [comment]
- resolved by 2deab5329167
